assets/script.js

```diff
@@ -0,0 +1,14 @@
+(function($) {
+	$('#billing_country_field').on('change', function() {
+
+		// Get country name
+		var country = $(this).find('#select2-billing_country-container').attr('title');
```

Comment:
Currently, this is getting the `title` string. It's not good at all if the site is another language. 

In this case, you should use the `value` string, for Vietnam, it would be `VN`.